core/arch/arm/kernel/user_ta.c

```diff
@@ -502,14 +501,28 @@ static const struct tee_ta_ops user_ta_ops __rodata_unpaged = {
 static SLIST_HEAD(uta_stores_head, user_ta_store_ops) uta_store_list =
 		SLIST_HEAD_INITIALIZER(uta_stores_head);
 
+/*
+ * Break unpaged attribute propagation to user_ta_ops structure
```

Comment:
I think it would be a little more clear with "unpaged attribute dependency propagation"

Reply:
ok, thanks.

Reply:
ok, thanks